packages/pwa-kit-react-sdk/package.json

```diff
@@ -49,14 +52,13 @@
   },
   "devDependencies": {
     "@loadable/component": "^5.15.3",
-    "@wojtekmaj/enzyme-adapter-react-17": "^0.8.0",
     "enzyme": "^3.11.0",
```

Comment:
This library does not declare peerDependency, it will be replaced in a different PR